.travis.yml

```diff
@@ -21,10 +21,10 @@ matrix:
       script: phpunit --bootstrap expandFns.php tests/phpunit
     - stage: deploy
       name: "Update WMFlabs servers"
-      if: (branch == master) AND (NOT env(TRAVIS_PULL_REQUEST))
-      script: bash <(echo wget -O- https://tools.wmflabs.org/citations/gitpull.php)
-      if: (branch == development) AND (NOT env(TRAVIS_PULL_REQUEST))
-      script: bash <(echo wget -O- https://tools.wmflabs.org/citations-dev/gitpull.php)
+      php: 5.6 # Avoid downloading and installing 5.5
+      script:
+        - if [ "$TRAVIS_BRANCH" = "master"      -a "$TRAVIS_PULL_REQUEST" = "false" ]; then wget -O- https://tools.wmflabs.org/citations/gitpull.php ; fi
```

Comment:
Will this still echo the output of the wget to the Travis log?
Nice to be able to see if there are any problems with the git pull on the server.

Reply:
I am checking that in another pull that has a version that always does the wget

Reply:
It does.  I just looked at the output of the other totally bogus pull